### PR TITLE
Adjust b3->b4 in docs, put date in changelog for release.

### PR DESCRIPTION
**Pull request by KieranBrantnerMagee**

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 7.0.0b4 (Unreleased)
+## 7.0.0b4 (2020-07-05)
```
Comment (yunhaoling) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:3]: haha, you're not going to do the release on Sunday

Comment (KieranBrantnerMagee) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md:3]: Unironically did this by adding days to today, I think I may have forgotten that the holiday weekend made today Friday.  Good call.
```suggestion
## 7.0.0b4 (2020-07-06)
```

**Comment by kristapratico** [comment]
This "Adjust b3->b4 in docs" caught my eye. Instead of updating b3 -> b4, you can use `latest` in the link. just FYI, either way works.

**Comment by KieranBrantnerMagee** [comment]
> 
> 
> This "Adjust b3->b4 in docs" caught my eye. Instead of updating b3 -> b4, you can use `latest` in the github.io link. just FYI, either way works.

Krista you just made my afternoon.  (I was thinking as I stubbed these out last night "this'll be painful to upkeep I wonder if there's a magic macro"; turns out more professional engineers than myself didn't just wonder about this XD)

**Pull request merged**